tests/end2end/features/downloads.feature

```diff
@@ -684,3 +684,16 @@ Feature: Downloading things from a website.
         When I set downloads.location.prompt to false
         And I open 500-inline
         Then the error "Download error: *INTERNAL SERVER ERROR" should be shown
+
+    ## External download path fileselector
+
+    Scenario: Select download path
+        When I set downloads.location.prompt to true
+        And I set fileselect.handler to external
+        And I setup a fake folder fileselector selecting "tests/end2end/data/backforward/" and writes to a temporary file
```

Comment:
```suggestion
        And I setup a fake folder fileselector selecting "(tmpdir)(dirsep)downloads(dirsep)subdir" and writes to a temporary file
```

to place the file into a well-defined directory which isn't the default. Search for `(tmpdir)` in this file for some inspiration - the subdir gets created by `Given I set up a temporary download dir` via the `download_dir` fixture in `test_downloads_bdd.py`.

Note that you will need to also adjust the Python code to carry out those replacements, as they are not done universally (thinking about it, maybe they should be?). See `run_command` in `tests/end2end/features/conftest.py` for inspiration.